packages/dev/core/src/XR/native/nativeXRFrame.ts

```diff
@@ -80,6 +80,10 @@ export class NativeXRFrame implements XRFrame {
     public readonly getImageTrackingResults = (): XRImageTrackingResult[] => {
         return this._nativeImpl._imageTrackingResults ?? [];
     };
+
+    public getDepthInformation(view: XRView): XRCPUDepthInformation | null {
+        return this._nativeImpl.getDepthInformation(view);

```

Comment:
CC @bghgary

Reply:
This function is not available in native unfortunately, so this won't work.

https://github.com/BabylonJS/BabylonNative/blob/master/Plugins/NativeXr/Source/NativeXr.cpp#L2100-L2110

Reply:
@drumath2237 - You can throw an exception here in the meantime, until the functionality is implemented in babylon native

Reply:
Thank you @bghgary and @RaananW .
I dealt with this like below.
```ts
    public getDepthInformation(view: XRView): XRCPUDepthInformation | null {
        throw new Error("This function is not available in Babylon Native");
        // return this._nativeImpl.getDepthInformation(view);
    }
```